### PR TITLE
GPooh and acute cup

### LINKED ISSUE #775: Bring in the `cup_marker` from the UCD to GP OoHs

### PR DESCRIPTION
**Pull request by lizihao-anu**
close #775 #776

### CODE REVIEW

File: R/read_extract_acute.R [hunk 1]
```diff
@@ -149,6 +149,7 @@ read_extract_acute <- function(year, file_path = get_boxi_extract_path(year = ye
       ipdc = "Inpatient Day Case Identifier Code",
       cij_ipdc = "CIJ Inpatient Day Case Identifier Code (01)",
       lineno = "Line Number (01)",
+      case_reference_number = "Case Reference Number [C]",
```
Comment (SwiftySalmon) [inline R/read_extract_acute.R:152]: Need to read in case ref no above this

Comment (SwiftySalmon) [reply R/read_extract_acute.R:152]: "Case Reference Number [C]" is missing from the boxi extract? where does this come from?

Comment (SwiftySalmon) [reply R/read_extract_acute.R:152]: nm. I was running a differnt year to you. Ran with 1920 and it works fine

File: R/add_activity_after_death_flag.R [hunk 2]
```diff
@@ -125,8 +125,8 @@ add_activity_after_death_flag <- function(
 #'
 #'
 # Read data------------------------------------------------
-process_deaths_lookup <- function(update = latest_update(),
-                                  write_to_disk = TRUE, ...) {
+process_combined_deaths_lookup <- function(update = latest_update(),
+                                           write_to_disk = TRUE, ...) {
```
Comment (SwiftySalmon) [inline R/add_activity_after_death_flag.R:129]: are these meant to be different? I do not want to overwrite changes on the September branch

Comment (lizihao-anu) [reply R/add_activity_after_death_flag.R:129]: > are these meant to be different? I do not want to overwrite changes on the September branch

Yeah. It's something that Jen and I agreed but she forgot to change. `process_combined_deaths_lookup` make more sense and is more intuitive. This function is used in `run_all_targets.R` and the new name `combined` is already in use.

**Review by SwiftySalmon — APPROVED** [review]
looks good to me

**Pull request merged**